anomalib/models/draem/config.yaml

```diff
@@ -0,0 +1,102 @@
+dataset:
+  name: mvtec #options: [mvtec, btech, folder]
+  format: mvtec
+  path: ./datasets/MVTec
+  category: bottle
+  task: segmentation
+  image_size: 256
+  train_batch_size: 8
+  test_batch_size: 32
+  num_workers: 8
+  transform_config:
+    train: ./anomalib/models/draem/transform_config.yaml
+    val: ./anomalib/models/draem/transform_config.yaml
+  create_validation_set: false
+  tiling:
+    apply: false
+    tile_size: null
+    stride: null
+    remove_border_count: 0
+    use_random_tiling: False
+    random_tile_count: 16
+
+model:
+  name: draem
+  anomaly_source_path: null # optional, e.g. ./datasets/dtd
```

Comment:
What is dtd? Does this make `dataset` section above obsolete?

Reply:
It's not the most easily digestible algorithm. It would probably be good to add a few sentences explaining this anomaly source dataset in the readme of the draem module.

Reply:
These are some examples of simulated anomalies, clearly showing the carried over textures of the original source images.

![Screenshot from 2022-06-03 16-46-15](https://user-images.githubusercontent.com/26067904/171877736-3aefb11f-25e1-447b-a75d-1d5c7322d521.png)
![Screenshot from 2022-06-03 16-45-59](https://user-images.githubusercontent.com/26067904/171877759-38290a25-78ea-41c2-bb89-2c171e29ee45.png)
![Screenshot from 2022-06-03 16-45-29](https://user-images.githubusercontent.com/26067904/171877781-e984bbe1-aff0-44af-933d-5649b1472ece.png)

Reply:
This is a bit similar to our algo in nous. 

https://scholar.google.com/citations?view_op=view_citation&hl=en&user=SVpL2VMAAAAJ&cstart=20&pagesize=80&citation_for_view=SVpL2VMAAAAJ:AHdEip9mkN0C